backend_new/test/integration/listing.e2e-spec.ts

```diff
@@ -572,4 +579,152 @@ describe('Listing Controller Tests', () => {
+      });
+
+      const listingData = await listingFactory(jurisdictionA.id, prisma);
+      listing = await prisma.listings.create({
+        data: listingData,
+      });
+      partnerUser = await prisma.userAccounts.create({
+        data: await userFactory({
+          roles: {
+            isPartner: true,
+            isAdmin: false,
+            isJurisdictionalAdmin: false,
+          },
+          listings: [listing.id],
+          jurisdictionId: jurisdictionA.id,
+        }),
+      });
+      const res = await request(app.getHttpServer())
+        .post('/auth/login')
+        .send({ email: adminUser.email, password: 'abcdef' })
+        .expect(201);
+
+      adminAccessToken = res.header?.['set-cookie'].find((cookie) =>
+        cookie.startsWith('access-token='),
+      );
+    });
+
+    it('update status to pending approval and notify appropriate users', async () => {
+      const val = await constructFullListingData(listing.id, jurisdictionA.id);
+      val.status = ListingsStatusEnum.pendingReview;
+      const putPendingApprovalResponse = await request(app.getHttpServer())
+        .put(`/listings/${listing.id}`)
+        .send(val)
+        .expect(200);
+
+      const listingPendingApprovalResponse = await request(app.getHttpServer())
+        .get(`/listings/${putPendingApprovalResponse.body.id}`)
+        .expect(200);
+
+      expect(listingPendingApprovalResponse.body.status).toBe(
+        ListingsStatusEnum.pendingReview,
+      );
+      expect(mockRequestApproval).toBeCalledWith(
+        undefined,
+        { id: listing.id, name: val.name },
+        expect.arrayContaining([adminUser.email, jurisAdmin.email]),
+        process.env.PARTNERS_PORTAL_URL,
+      );
+      //ensure juris admin is not included since don't have approver permissions in alameda seed
+      expect(mockRequestApproval.mock.calls[0]['emails']).toEqual(
+        expect.not.arrayContaining([wrongJurisAdmin.email, partnerUser.email]),
+      );
+    });
+
+    it('update status to listing approved and notify appropriate users', async () => {
+      const val = await constructFullListingData(listing.id, jurisdictionA.id);
+      val.status = ListingsStatusEnum.active;
+      const putApprovedResponse = await request(app.getHttpServer())
+        .put(`/listings/${listing.id}`)
+        .send(val)
+        .expect(200);
+
+      const listingApprovedResponse = await request(app.getHttpServer())
+        .get(`/listings/${putApprovedResponse.body.id}`)
+        .expect(200);
+
+      expect(listingApprovedResponse.body.status).toBe(
+        ListingsStatusEnum.active,
+      );
+      expect(mockListingApproved).toBeCalledWith(
+        undefined,
+        { id: listing.id, name: val.name },
+        expect.arrayContaining([partnerUser.email]),
+        jurisdictionA.publicUrl,
+      );
+    });
+
+    // Skipping while trying to figure out req.user
+    it.skip('update status to changes requested and notify appropriate users', async () => {
```

Comment:
should this still be skipped?

Reply:
Updated the code to accomodate the login flow again. So this is now working and the other tests are actually work now as well